test/resolver.spec.js

```diff
@@ -77,19 +78,36 @@ describe('IPLD format resolver (local)', () => {
 
         expect(paths).to.eql([
           'name',
+          'nest',
```

Comment:
The one that was before, was listing the paths of a Buffer as if it was an array, listing a path per byte of the Buffer.

This new test verifies that the things that are listed, are the ones that are resolvable, given the root of the graph. The best way is just to open the object, since it is small, and just list out all the paths yourself to verify it is correct.